examples/todos-flow/src/components/todo-list.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import React from 'react';
+
+import Todo from './todo';
+
+import type { Todos, Id } from '../types/todos';
+
+export type Props = {
+  todos: Todos,
+  onTodoClick: (id: Id) => void
+};
+
+const TodoList = ({ todos, onTodoClick }: Props) => (
+  <ul>
+    {todos.map(todo => (
+      <Todo key={todo.id} {...todo} onClick={() => onTodoClick(todo.id)} />
+    ))}
+  </ul>
+);
+
+export default TodoList;
```

Comment:
BTW, this file should be named `TodoList.js` to match the component name that it exports.